css/layout.css

```diff
@@ -286,3 +288,154 @@ summary {
 .form .button {
   padding: 20px 30px;
 }
+
+@media screen and (min-width: 992px) {

```

Comment:
- [ ] I admit that your project looks amazing both mobile and desktop 👍. However, I can see that you specified your desktop version breaking point to be `@media screen and (min-width: 992px)` which is a good desktop breaking point. But I can see that the same breaking point is been used more than once in your project. Please consider moving all the styles under this breaking point to one place instead of repeating the same breaking point over and over.